pkg/asset/ignition/machine/master.go

```diff
@@ -93,3 +112,42 @@ func (a *Master) Load(f asset.FileFetcher) (found bool, err error) {
 	a.File, a.Config = file, config
 	return true, nil
 }
+
+func AddEtcdDisk(config *igntypes.Config, device string) {
```

Comment:
@yuqi-zhang @patrickdillon suggested that I get your opinion on these changes. When you have a moment can you take a look?

We are starting an enhancement too but it isn't ready for a PR just yet.

Reply:
https://github.com/openshift/enhancements/pull/1805